protocol/x/vault/keeper/msg_server_allocate_to_vault.go

```diff
@@ -29,40 +26,8 @@ func (k msgServer) AllocateToVault(
 		)
 	}
 
-	// Check if vault has a corresponding clob pair.
-	_, exists := k.Keeper.clobKeeper.GetClobPair(ctx, clobtypes.ClobPairId(msg.VaultId.Number))
-	if !exists {
-		return nil, types.ErrClobPairNotFound
-	}
-
-	// If vault doesn't exist:
-	// 1. initialize params with `STAND_BY` status.
-	// 2. add vault to address store.
-	_, exists = k.Keeper.GetVaultParams(ctx, msg.VaultId)
-	if !exists {
-		err := k.Keeper.SetVaultParams(
-			ctx,
-			msg.VaultId,
-			types.VaultParams{
-				Status: types.VaultStatus_VAULT_STATUS_STAND_BY,
-			},
-		)
-		if err != nil {
-			return nil, err
-		}
-		k.Keeper.AddVaultToAddressStore(ctx, msg.VaultId)
-	}
-
-	// Transfer from main vault to the specified vault.
-	if err := k.Keeper.sendingKeeper.ProcessTransfer(
-		ctx,
-		&sendingtypes.Transfer{
-			Sender:    types.MegavaultMainSubaccount,
-			Recipient: *msg.VaultId.ToSubaccountId(),
-			AssetId:   assetstypes.AssetUsdc.Id,
-			Amount:    msg.QuoteQuantums.BigInt().Uint64(), // validated to be positive above.
-		},
-	); err != nil {
+	err := k.Keeper.TransferToVault(ctx, msg.VaultId, msg.QuoteQuantums.BigInt())
```

Comment:
nit: should we just call this keeper method `AllocateToVault`?

Reply:
I just called this what it was specified in the design doc